man/man8/zfs.8

```diff
@@ -1107,16 +1123,26 @@ This property is not available on volumes, on file systems before version 4, or
 .RE
 Files created on Linux always have POSIX owners.
 
+.RS 4
+The \fBuserobjquota\fR is similar to \fBuserquota\fR but it limits that the \fIuser\fR can consume \fIcount\fR number of objects at most.
+Please refer to \fBuserobjused\fR for more information about how zfs counts object usage.
```

Comment:
How about:

The \fBuserobjquota\fR is similar to \fBuserquota\fR but it limits the number of objects a \fIuser\fR can create.
Please refer to \fBuserobjused\fR for more information about how ZFS counts object usage.